### DISCUSSION TITLE
Difference in output of transformation when using XML and XSD.

### DISCUSSION THREAD

**Discussion by abhishekvishnoi** [atlasmap/atlasmap#3487] [Q&A]
While transforming from a json to SOAP request .  

I have tried both ways for importing and out put 
	ie : 1. using xml
	       2. using created from the same xml as in step 1.
The transformation remains same , however there is significant difference in the structure of the both scenarios. (refer imag

Steps to re produce :
1.  from zip file : soap_namespace_and_structure_issue_with_xml_instance , get ns2_input_json.json as source .
2. get ns2_output_xml_instance.xml as target (choose instance while importing)
3. refer the .adm file in the zip file ,
4.  I have attached the expected and generated xmls.

5.  frim zipfile soap_namespace_issue_with_xml_schema.zip , get ns2_input_json.json as source .
6. get ns2_output_xml_schema.xsd
7. refer the .adm file in the zip file ,
8.  I have attached the expected and generated xmls.

both mappings generate the output , however compaing the output generate from mapping int step3 and mapping from step8 are different in structure.

[soap_namespace_and_structure_issue_with_xml_instance.zip]
(https://github.com/atlasmap/atlasmap/files/7551978/soap_namespace_and_structure_issue_with_xml_instance.zip)
e
[soap_namespace_issue_with_xml_schema.zip](https://github.com/atlasmap/atlasmap/files/7551980/soap_namespace_issue_with_xml_schema.zip)
![Screen_Capture_-_16_Nov__2_01_pm](https://user-images.githubusercontent.com/50903850/142140615-6f7159c4-0b22-494f-a4fe-30452bc721be.png)

)

**Comment by igarashitm** [comment]
Creating AtlasMap document from XML instance is, **assuming XML structure without schema**. There could be always ambiguity on it. If you have XML schema, it's better using XML schema.
